File: BehaviorPattern/Command/Golang/Concretes/LightOffCommand.go

package concretes

import (
	"v/Receivers"
	"v/Interfaces"
)

/**
 * Class Light Off Command implementation Command Interface
 */
type LightOffCommand struct {
	Light receivers.Light
}

// Constructor
func NewLightOffCommand(light receivers.Light) interfaces.Command {
	return &LightOffCommand{Light: light}
}

//  Override Execute command
func (l *LightOffCommand) Execute() {
	l.Light.Off()
}

File: BehaviorPattern/Command/Python/Concretes/LightOffCommand.py

from Interfaces import Command
from Receivers import Light

class LightOffCommand(Command):
    """
        Class Light Off Command implementation Command Interface
    """
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.light: Light = kwargs.get('light', None)
        if args:
            self.light = args[0]
            
    # Override
    def execute(self):
        """
            Execute command
        """
        self.light.off()